py-polars/polars/dependencies.py

```diff
@@ -183,6 +187,8 @@ def _lazy_import(module_name: str) -> tuple[ModuleType, bool]:
     )
     deltalake, _DELTALAKE_AVAILABLE = _lazy_import("deltalake")
     fsspec, _FSSPEC_AVAILABLE = _lazy_import("fsspec")
+    holoviews, _HOLOVIEWS_AVAILABLE = _lazy_import("holoviews")
```

Comment:
Do we still need the holoviews import/configurations now that we use the `post_patch` approach?

Reply:
good one thanks